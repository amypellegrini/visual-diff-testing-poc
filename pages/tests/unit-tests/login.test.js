import Login from "../../login";
import React from "react";
import { render } from "react-testing-library";

describe("Login page", () => {
  test("renders", () => {
    const { container } = render(<Login />);
    expect(container).toMatchSnapshot();
  });
});
