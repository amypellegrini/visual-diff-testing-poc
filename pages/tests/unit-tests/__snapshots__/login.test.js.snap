// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page renders 1`] = `
<div>
  <div>
    <p>
      Login page
    </p>
  </div>
</div>
`;
